manifest.json

```diff
@@ -1905,6 +1905,7 @@
       "react-native-video@4.4.5",
       "react-native-view-overflow@0.0.5",
       "react-native-view-shot@3.1.2",
+      "react-native-webview@11.26.1",
```

Comment:
Most likely this should work with at least a few prior version as well, right?
If yes, can you add the manifest entry to 0.45-0.49 too?

Reply:
cool. But I have added the plugin in 0.46, should I move it to 0.45 and add support from 0.45-0.49 or just 0.46-0.49 is good?

Reply:
done